suites/selftest/ts/minimal/multi_line_indent.c

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright (C) 2025 OKTET Ltd. All rights reserved. */
+/** @file
+ * @brief Minimal test
+ *
+ * Minial test scenario that checks indentation stripping.
+ */
+
+/** @page minimal_multi_line_indent Multi-line indentation test
+ *
+ * @objective Check that indentation of multi-line is stripped correctly by TE.
+ *
+ * @par Test sequence:
+ *
+ */
+
+#ifndef DOXYGEN_TEST_SPEC
+
+/** Logging subsystem entity name */
+#define TE_TEST_NAME    "multi_line_indent"
+
+#include "te_config.h"
+#include "tapi_test.h"
+
+int
+main(int argc, char **argv)
+{
+    const char *test_line;
+    TEST_START;
+
+    TEST_GET_STRING_PARAM(test_line);
+
+    TEST_STEP("Print the value of 'test_line' parameter");
+    RING("Value of 'test_line11' param is: %s", test_line);
+
+    TEST_SUCCESS;
```

Comment:
IMHO we should really test what happens. Yes, it will duplicate the parameter value, but it will make the test not dummy.